packages/react/src/themes/teams/components/Toolbar/toolbarMenuStyles.ts

```diff
@@ -2,12 +2,12 @@ import { ICSSInJSStyle } from '@fluentui/styles'
 import { getColorScheme } from '../../colors'
 
 const toolbarMenuStyles = {
-  root: ({ props: p, variables: v }): ICSSInJSStyle => {
+  root: ({ props: p, variables: v, theme: { siteVariables } }): ICSSInJSStyle => {
```

Comment:
Let's not use directly `siteVariables` in the styles. Set this into some variable inside the toolbar (`toolbarMenuZIndex` for example) we will still be using the same values, but we won't access directly the `siteVariables`